galaxy_ng/app/models/aiindex.py

```diff
@@ -0,0 +1,32 @@
+from django.core.validators import RegexValidator
+from django.db import models
+
+SCOPES = (
+    ("namespace", "Namespace"),
+    ("legacy_namespace", "Legacy Namespace"),
+)
+
+
+namespace_validator = RegexValidator(
```

Comment:
should probably add a comment since Namespace and LegacyNamespace use different regex patterns, maybe:
```
# this regex is the most restrictive pattern that applies to both Namespace and LegacyNamespace
```